README.md

```diff
@@ -10,8 +10,12 @@ It was bootstrapped with [Create React App](https://github.com/facebookincubator
 
 ### Prerequisites
 
-To use the dashboards-app in development mode, it is necessary to have a running DHIS2 instance, and be logged in to it. This is because the app requests resources, like react and react-dom, from the DHIS2 core resources. Most developers use their local DHIS2 instance. If you haven't configured your DHIS2_HOME env variable and set up a config.json, then the app will default to using localhost:8080 with the admin user (see
-[config/webpack.config.dev.js](config/webpack.config.dev.js#L35)).
+To use the dashboards-app in development mode, it is necessary to have a running DHIS2 instance, and be logged in to it. This is because the app requests resources, like maps plugin, from the DHIS2 core resources. Most developers use their local DHIS2 instance.
+
+The following env vars need to be set when running in dev mode:
+
+-   REACT_APP_DHIS2_BASE_URL (e.g., http://localhost:8080)
+-   REACT_APP_DHIS2_AUTHORIZATION (base64 encoded username and password)
```

Comment:
I don't like requiring this to be an environment variable, isn't it better to just rely on cookies and manual login?  Otherwise you probably end up only testing with a single user.

Reply:
The analytics plugins need the password in order to work (in dev mode), unfortunately

Reply:
@jenniferarnesen why is that?

Reply:
I'm not sure - Jan would know. But my testing confirms that the plugins don't work without a password.